natives.json

```diff
@@ -33521,16 +33521,16 @@
 			"build": "1311"
 		},
 		"0xA6AA9F56BC6CFF58": {
-			"name": "_0xA6AA9F56BC6CFF58",
+			"name": "INVENTORY_ENABLE_MISSION_INVENTORY",
```

Comment:
Forgot to rename this one? This is the script function's name. I think it's probably something like ``INVENTORY_USE_MISSION_INVENTORY`` (don't forget the underscore prefix).